enterprise/models.py

```diff
@@ -2508,3 +2508,44 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class EnterpriseAnalyticsUser(TimeStampedModel):
+    """
+    Model for analytics users.
+
+     .. no_pii:
+    """
+    enterprise_customer_user = models.ForeignKey(
+        EnterpriseCustomerUser,
+        blank=False,
+        null=False,
+        on_delete=models.CASCADE
+    )
+    analytics_user_id = models.CharField(
```

Comment:
should we need to record the name of `third party analytics system` to know the service user belongs to?

Reply:
For the foreseeable future it is Tableau. Once we shift to another system, we can always add a column with the name, if required.